<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VpnOptionsWidget</class>
 <widget class="QWidget" name="VpnOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="dialingGroupBox">
     <property name="title">
      <string>Dialing optopns:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="displayProgressCheckBox">
        <property name="text">
         <string>Display progress while connecting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="promptCheckBox">
        <property name="text">
         <string>Prompt for name and password, certificate, etc.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="includeCheckBox">
        <property name="text">
         <string>Include Windows logon domain</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="redialingOptionsGroupBox">
     <property name="title">
      <string>Redialing options:</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="redialAttemptsLabel">
        <property name="text">
         <string>Redial attemtps:</string>
        </property>
        <property name="buddy">
         <cstring>redialAttemptsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="redialAttemptsSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>Time between redial attempts:</string>
        </property>
        <property name="buddy">
         <cstring>timeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="timeComboBox">
        <item>
         <property name="text">
          <string>1 second</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30 seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 minute</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 minutes</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="idleTimeLabel">
        <property name="text">
         <string>Idle time before handing up:</string>
        </property>
        <property name="buddy">
         <cstring>idleTimeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="idleTimeComboBox">
        <item>
         <property name="text">
          <string>never</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 minute</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24 hours</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="redailCheckBox">
        <property name="text">
         <string>Redial if time is dropped</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>123</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>displayProgressCheckBox</tabstop>
  <tabstop>promptCheckBox</tabstop>
  <tabstop>includeCheckBox</tabstop>
  <tabstop>redialAttemptsSpinBox</tabstop>
  <tabstop>timeComboBox</tabstop>
  <tabstop>idleTimeComboBox</tabstop>
  <tabstop>redailCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
